temba_client/tests.py

```diff
@@ -23,13 +23,17 @@ class TembaTest(unittest.TestCase):
     """
     API_VERSION = None
 
-    def read_json(self, filename):
+    def read_json(self, filename, extract_result=None):
```

Comment:
Saves having separate test files for responses from POSTs - this way we can just extract a single result from a test file containing several results